change/@ni-nimble-angular-e0a1c358-b227-471d-8b20-44d168794738.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
nit: before merging the PR comment should be updated with where we end up

Reply:
Updated the PR description